jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -85,6 +85,16 @@
           >
           </v-text-field>
         </v-row>
+
+        <v-row v-if="trace_type_selected==='Auto'">
+          <v-select
+            :items="trace_peak_method_items"
+            v-model="trace_peak_method_selected"
+            label="Peak Method"
+            hint="Method to use for identifying the peak in each bin"
```

Comment:
Can we be any more specific in the hint, or is "peak" enough on its own? I'm thinking about alternatives like "peak flux" (not always correct) or "peak signal."

Reply:
"peak pixel"?